Sources_v3/StreamChatUI/ChatChannel/Reactions/ChatMessageReactionsBubbleView.swift

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatMessageReactionsBubbleView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
+    public var content: Content? {
+        didSet { updateContentIfNeeded() }
+    }
+
+    // MARK: - Subviews
+
+    public private(set) lazy var contentView = uiConfig
+        .messageList
+        .messageReactions
+        .reactionsView
+        .init()
+        .withoutAutoresizingMaskConstraints
+
+    open var tailLeadingAnchor: NSLayoutXAxisAnchor { contentView.centerXAnchor }
+    open var tailTrailingAnchor: NSLayoutXAxisAnchor { contentView.centerXAnchor }
```

Comment:
Don't like this part since `ChatMessageReactionsBubbleView` doesn't have a tail but knows about it. Those anchors are required to make the tail position dynamic.